contracts/FlowIDTableStaking.cdc

```diff
@@ -803,8 +803,15 @@ pub contract FlowIDTableStaking {
             }
         }
 
+        pub fun changeMinimumStakeRequirements(_ newRequirements: {UInt8: UFix64}) {
+            pre {
+                newRequirements.keys.length == 5: "Incorrect number of nodes"
+            }
+            FlowIDTableStaking.minimumStakeRequired = newRequirements
+        }
+
         // Changes the total weekly payout to a new value
-        pub fun updateEpochTokenPayout(_ newPayout: UFix64) {
+        pub fun changeEpochTokenPayout(_ newPayout: UFix64) {
```

Comment:
I favour personally favour "set" for setters.

Reply:
I like that too. I'll change it